pkgs/native_assets_cli/test/model/asset_test.dart

```diff
@@ -20,45 +20,30 @@ void main() {
       name: 'foo',
       file: fooUri,
       linkMode: DynamicLoadingBundled(),
-      os: OS.android,
-      architecture: Architecture.x64,
     ),
     CodeAsset(
       package: 'my_package',
       name: 'foo3',
       linkMode: DynamicLoadingSystem(foo3Uri),
-      os: OS.android,
-      architecture: Architecture.x64,
     ),
     CodeAsset(
       package: 'my_package',
       name: 'foo4',
       linkMode: LookupInExecutable(),
-      os: OS.android,
-      architecture: Architecture.x64,
-    ),
-    CodeAsset(
-      package: 'my_package',
-      name: 'foo5',
-      linkMode: LookupInProcess(),
-      os: OS.android,
-      architecture: Architecture.x64,
     ),
+    CodeAsset(package: 'my_package', name: 'foo5', linkMode: LookupInProcess()),
     CodeAsset(
       package: 'my_package',
       name: 'bar',
       file: barUri,
-      os: OS.linux,
-      architecture: Architecture.arm64,
+
```

Comment:
Same here and in other places of this file - I would have expected the formatter to catch and correct this? I am surprised that no check turned red?

Reply:
I see, I'd expect no blank lines. (This is how the previous formatter worked?) e.g. if I regex remove an argument from the list I wasn't including the new lines before.

Reply:
> (This is how the previous formatter worked?)

Yeah, the old formatting style made a bigger distinction between how argument lists and collection literals were formatted. The former didn't allow newlines but the latter did. In the new formatter, since both are formatted in a similar style, they reuse a lot of the same formatting logic and behave more similarly, including allowing newlines inside them.